src/argilla/server/telemetry.py

```diff
@@ -64,9 +67,14 @@ def __post_init__(self, enable_telemetry: bool, disable_send: bool, api_key: str
             "platform": platform.platform(),
             "python_version": platform.python_version(),
             "sys_version": platform.version(),
+            "deployment": "quickstart" if is_quickstart_server() else "server",
```

Comment:
If the function is not used for other things maybe you can changed it to return an string with the type of server something like:

```suggestion
            "deployment": server_type(),
```